container.go

```diff
@@ -206,3 +224,11 @@ func getCheckpointSize(path string) (size int64, err error) {
 
 	return dirSize(dir)
 }
+
+func shortenPath(path string) string {
```

Comment:
I am not sure how I feel about this. I understand why it might be nice for very long paths, but I think it should be optional. Maybe default to this output, but also provide a way to display the full paths.